### PR TITLE
Revert free to paid cross currency upgrades

### PR DESCRIPTION
**Pull request by tomverran**

### CODE REVIEW

**Comment by AWare** [comment]
THEATRE!

File: frontend/app/services/MemberService.scala [hunk 1]
```diff
@@ -158,25 +158,20 @@ class MemberService(identityService: IdentityService,
...
     (for {
-      sub <- EitherT(subOrPendingAmendError(friend.subscription))
-      customer <- stripeService.Customer.create(friend.contact.identityId, form.payment.token).liftM
-      paymentResult <- createPaymentMethod(friend.contact, customer).liftM
-      subRes <- createPaidSubscription(friend.contact,form,NameForm(friend.contact.firstName.getOrElse(""),friend.contact.lastName),newTier,customer,code).liftM
-      _ <- salesforceService.updateMemberStatus(IdMinimalUser(friend.contact.identityId, None), newTier, Some(customer)).liftM
-      _ <- zuoraService.cancelPlan(sub, DateTime.now.toLocalDate).liftM
-    } yield {
-      form.addressDetails.foreach(identityService.updateUserFieldsBasedOnUpgrade(friend.contact.identityId, _))
-      track(MemberActivity("upgradeMembership", MemberData(
```
Comment (paulbrown1982) [inline frontend/app/services/MemberService.scala]: Do we still need this even after reverting?

Comment (tomverran) [inline frontend/app/services/MemberService.scala]: This happens in ```upgradeSubscription```

**Comment by AWare** [comment]
👍

**Comment by mario-galic** [comment]
Fail-in-PROD seems to be genuine. It might be related to [this issue](https://app.getsentry.com/the-guardian/identity-frontend-prod/issues/127847329/):

```
User could not be logged in after registering: List(com.gu.identity.frontend.errors.SignInServiceGatewayAppException: Something went wrong)
```

The test user's account was indeed created (ID 16161440), but automatic sign in failed.

**Pull request merged**